Backend/Remora.Discord.API.Abstractions/API/Gateway/Events/Invites/IInviteCreate.cs

```diff
@@ -76,7 +76,12 @@ public interface IInviteCreate : IGatewayEvent
         /// <summary>
         /// Gets the type of user target for this invite.
         /// </summary>
-        Optional<InviteTarget> TargetUserType { get; }
+        Optional<InviteTarget> TargetType { get; }
```

Comment:
Try to keep these in the same order as they appear in the Discord docs (that is, target_type first and target_user after).

Reply:
Fixed, thanks